script.js

```diff
@@ -1,4 +1,128 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// Game states
+//var gameState_diceCount = "gameState_diceCount";
+var gameState_diceRoll = "gameState_diceRoll";
+var gameState_diceOrder = "gameState_diceOrder";
+
+// Dice count tracker
+// var numDiceCount = 0;
+
+// Initial game state
+var gameMode = gameState_diceRoll;
+
+// Player 1 starts
+var currPlayer = 1;
+
+// Dice roll results tracker as an array
+var player1Rolls = [];
+var player2Rolls = [];
+
+// Dice simulation
+var getDiceRoll = function () {
+  return Math.ceil(Math.random() * 6);
+};
+var getDiceRoll2 = function () {
+  var twoDiceRolled = [getDiceRoll(), getDiceRoll()];
+  // Insert round results to player results tracker array
+  if (currPlayer === 1) {
+    player1Rolls = twoDiceRolled;
+  }
+  if (currPlayer === 2) {
+    player2Rolls = twoDiceRolled;
+  }
+  return twoDiceRolled;
+};
+
+var concatenate2Numbers = function (num1, num2) {
+  return Number(String(num1) + String(num2));
+};
+
+var playerNumber = function () {
+  var diceArray;
+  if (currPlayer === 1) {
+    diceArray = player1Rolls;
+  }
+  if (currPlayer === 2) {
+    diceArray = player2Rolls;
+  }
+  return diceArray;

```

Comment:
Because you wrote a return statement in line 47, your code will always exit from the `playerNumber` function. This means that lines 48 to 59 will never run. Try removing the return so you will be able to run the code below.